README.md

```diff
@@ -2,6 +2,30 @@
 
 Click here to view Live [Magic-books](https://tanzila-magic-books.herokuapp.com/).
 
+  
+## Demo
+
+[Click here to view the live demo](https://bookstore-rb.herokuapp.com/)
```

Comment:
This is different from your current app version when I run it on my local saver kindly check to correct to the latest version of your app

Reply:
Hello Taiwo,

I don't think it's required for us to update Heroku every time we complete a milestone . I will do so for my last milestone.

Reply:
Hi @tanzila-abedin , I requested for this because you have two live links which contradict,one is not your app current version while the other is another app version and your readme file is not descriptive on what your magic book project is all about. Sorry for any inconvenience.

Reply:
Hi @taiwo2 
I updated the readme with necessary changes but I am failing to see any prop error from my end. Kindly send an ss if you can still see it so I know what to look for. Thank you :)

Reply:
I commented on the props not showing error, not that it's not working. I Highlighted it to commend your work and effort on it